go/cmd/dolt/commands/status.go

```diff
@@ -16,22 +16,49 @@ package commands
 
 import (
 	"context"
+	"encoding/json"
 	"errors"
 	"fmt"
-	"io"
+	"github.com/dolthub/dolt/go/libraries/doltcore/ref"
+	"strconv"
+	"strings"
 
-	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
-	"github.com/dolthub/dolt/go/libraries/doltcore/env/actions/commitwalk"
-	"github.com/dolthub/dolt/go/store/hash"
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/fatih/color"
 
 	"github.com/dolthub/dolt/go/cmd/dolt/cli"
 	"github.com/dolthub/dolt/go/cmd/dolt/errhand"
-	"github.com/dolthub/dolt/go/libraries/doltcore/diff"
+	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
 	"github.com/dolthub/dolt/go/libraries/doltcore/env"
-	"github.com/dolthub/dolt/go/libraries/doltcore/merge"
 	"github.com/dolthub/dolt/go/libraries/utils/argparser"
 )
 
+type printData struct {
```

Comment:
Brief godoc here would help the reader know the purpose of this structure. Not essential, but when combined with the undocuments createPrintData function, I'm forced to infer that there is a stage where you read all the essential data, then follow that up with printing all the data.

I know about this staging because of comments in the Exec method. My style is to flip this on it's head, and put most comments outside of the code on structure and function definitions. This seems to be the pattern followed on the dolt code I've been exposed to